policy/modules/system/systemd.te

```diff
@@ -875,6 +885,28 @@ optional_policy(`
 ')
 
 
+#######################################
+#
+# Rfkill local policy
+#
+
+manage_dirs_pattern(systemd_rfkill_t, systemd_rfkill_var_lib_t, systemd_rfkill_var_lib_t)
+manage_files_pattern(systemd_rfkill_t, systemd_rfkill_var_lib_t, systemd_rfkill_var_lib_t)
+
+dev_read_sysfs(systemd_rfkill_t)
+dev_rw_wireless(systemd_rfkill_t)
+
+init_var_lib_filetrans(systemd_rfkill_t, systemd_rfkill_var_lib_t, dir, "rfkill")
```

Comment:
Since this does not create other `var_lib_t` or `*_var_lib_t` directories, the `"rfkill"` name does not seem necessary.

This line should go up with the `manage_*_pattern` lines.